science/projects/zahrasadat_sajjadi/01-main-content/zarebin_p1.ipynb

```diff
@@ -0,0 +1,2717 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.                                  content_list.append(str(content)) # example -&gt; content: hello world!
</pre><p>each feature could have a separate function for extraction (for code readability and stuff :D)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>